docker/docker-compose.yml

```diff
@@ -14,6 +14,11 @@ services:
       - DB_USER=${MONGODB_USER}
       - DB_PASSWORD=${MONGODB_PASSWORD}
       - DB_PORT=${MONGODB_PORT}
+      - SMTP_HOST=${SMTP_HOST}
+      - SMTP_PORT=${SMTP_PORT}
+      - SMTP_USERNAME=${SMTP_USERNAME}
+      - SMTP_PASS=${SMTP_HOST}
```

Comment:
These two parameters are not necessary for our SMTP server, so they can be removed.